src/PillowResize/PillowResize.cc

```diff
@@ -16,7 +16,7 @@
 
 #include <cstdint>
 #include <utility>
-
+#include <memory>
```

Comment:
I think it's better to add this include in the header file since it's missing also there

Reply:
yes, that's right.